docs/concepts_about_system_analysis.md

```diff
@@ -0,0 +1,53 @@
+# Concepts about System Analysis
+In this docs, we will explain some important concepts that
+you need to know before performing system analyses using Explainaboard.
+
+## What a "dataset" and "system output" is
+To perform an analysis of your results, usually, two types of files should be prepared: `dataset`
```

Comment:
```suggestion
To perform an analysis of your results, usually, two types of files should be prepared: "dataset"
```

I think that `dataset` should mainly be used when "dataset" indicates a piece of code or a variable, which is not the case here. Probably just double quotes is fine? Similarly below.